src/main/java/com/example/enterpriseapplication/RecipeEaseController.java

```diff
@@ -15,6 +15,17 @@ public class RecipeEaseController {
     public String index(){
         return "home";
     }
+
+    @RequestMapping("/addRecipe")
+    public String addRecipe(){
+        return "AddRecipe";
+    }
+
+    @RequestMapping("/recipeAdded")
+    public String recipeAdded(){
+        return "RecipeAdded";
+    }
```

Comment:
Here, I added the missing mappings to your other template files. In addition to this, I updated the links in your application to reference these new routes.